src/graphene-rect.h

```diff
@@ -49,6 +49,15 @@ GRAPHENE_BEGIN_DECLS
 #define GRAPHENE_RECT_INIT(_x,_y,_w,_h) \
   (graphene_rect_t) { .origin = { .x = (_x), .y = (_y) }, .size = { .width = (_w), .height = (_h) } }
 
+/**
+ * GRAPHENE_RECT_INIT_ZERO:
+ *
+ * Initializes a #graphene_rect_t to (0, 0, 0, 0) when declaring it.
```

Comment:
I'd use "Initializes a #graphene_rect_t to a degenerate rectangle with an origin in (0, 0) and a size of 0", which is the term we use in `graphene_rect_intersection()`.

Reply:
Done